server start success,but login error,caused by NullPointerException

latest commit : b887fad0da852d2a845d2bf7ab7a9644fcc52a54

`[ERROR] 16:02:21.128 [http-nio-8088-exec-8] o.a.c.c.C.[.[.[.[dispatcherServlet] - Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception
java.lang.NullPointerException: null
        at org.apache.shardingsphere.elasticjob.lite.ui.security.UserAuthenticationService.checkUser(UserAuthenticationService.java:65)
        at org.apache.shardingsphere.elasticjob.lite.ui.security.AuthenticationFilter.handleLogin(AuthenticationFilter.java:76)
        at org.apache.shardingsphere.elasticjob.lite.ui.security.AuthenticationFilter.doFilter(AuthenticationFilter.java:58)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)`

![image](https://user-images.githubusercontent.com/24759691/178683393-d96847c9-c1c1-4846-9e36-762b3426f043.png)

--- a/shardingsphere-elasticjob-ui-distribution/shardingsphere-elasticjob-cloud-ui-bin-distribution/src/main/resources/application.properties
+++ b/shardingsphere-elasticjob-ui-distribution/shardingsphere-elasticjob-cloud-ui-bin-distribution/src/main/resources/application.properties
@@ -17,10 +17,8 @@
 
 server.port=8088
 
-auth.root_username=root
-auth.root_password=root
-auth.guest_username=guest
-auth.guest_password=guest
+auth.username=root
+auth.password=root
 auth.token_expires_after_seconds=3600
 
 # ElasticJob-Cloud's zookeeper address

--- a/shardingsphere-elasticjob-ui-distribution/shardingsphere-elasticjob-lite-ui-bin-distribution/src/main/resources/application.properties
+++ b/shardingsphere-elasticjob-ui-distribution/shardingsphere-elasticjob-lite-ui-bin-distribution/src/main/resources/application.properties
@@ -17,10 +17,8 @@
 
 server.port=8088
 
-auth.root_username=root
-auth.root_password=root
-auth.guest_username=guest
-auth.guest_password=guest
+auth.username=root
+auth.password=root
 auth.token_expires_after_seconds=3600
 
 spring.datasource.default.driver-class-name=org.h2.Driver
@@ -28,3 +26,6 @@ spring.datasource.default.url=jdbc:h2:mem:
 spring.datasource.default.username=sa
 spring.datasource.default.password=
 spring.jpa.show-sql=false
+
+## Uncomment the following property to allow adding DataSource dynamically.
+# dynamic.datasource.allowed-driver-classes={'org.h2.Driver','org.postgresql.Driver'}

